src/main/java/by/mifort/automation/hr/dev/util/impl/MappingDtoComponentConverterImpl.java

```diff
@@ -0,0 +1,60 @@
+package by.mifort.automation.hr.dev.util.impl;
+
+import by.mifort.automation.hr.dev.dto.CommunicationHistoryDto;
+import by.mifort.automation.hr.dev.dto.CandidateDto;
+import by.mifort.automation.hr.dev.entity.CommunicationHistory;
+import by.mifort.automation.hr.dev.entity.Keyword;
+import by.mifort.automation.hr.dev.entity.Candidate;
+import by.mifort.automation.hr.dev.util.MappingDtoComponentConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class MappingDtoComponentConverterImpl implements MappingDtoComponentConverter {
+    @Override
+    public CandidateDto convertToPeopleDto(Candidate candidate) {
+        CandidateDto candidateDto = new CandidateDto();
+        candidateDto.setId(candidate.getId());
+        candidateDto.setStatus(candidate.getStatus());
+        candidateDto.setLastContact(candidate.getLastContact());
+        if(candidate.getCandidateKeywords() != null)
```

Comment:
please use code formatting, and curly braces for if-else statement

Reply:
fixed